src/components/generic/Icon/Icon.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Icon as MomentumIcon} from '@momentum-ui/react';
+import {
+  CameraIcon,
+  AudioMicrophoneIcon,
+  AudioMicrophoneMutedIcon,
+  SettingsIcon,
+  ShareScreenIcon,
+  CancelIcon,
+  CameraMutedIcon,
+} from '../../icons';
+
+/**
+ * Displays an Icon based on its name.
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} props.name  Icon name
+ * @param {number} [props.size=24]  Icon width and height
+ * @returns {object} JSX of the Icon
+ *
+ */
+export default function Icon({name, size}) {
+  const baseName = name.split('_')[0];
+
+  const icons = {
```

Comment:
this can be defined outside of the function (above)

If possible, let's have all keys quoted